components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -729,4 +748,243 @@ TEST_F(BravePrefProviderTest, EnsureNoWildcardEntries) {
+  httpse_settings.CheckSettingsAreDefault(url);
+
+  // Verify that adblock only mode doesn't override settings.
+  testing_profile()->GetPrefs()->SetBoolean(
+      brave_shields::prefs::kAdblockAdBlockOnlyModeEnabled, true);
+  enabled_settings.CheckSettingsAreDefault(url);
+  httpse_settings.CheckSettingsAreDefault(url);
+
+  provider.ShutdownOnUIThread();
+}
+
+TEST_F(BravePrefProviderTest,
+       AdblockOnlyModeContentSettings_UserDefinedValues) {
+  base::test::ScopedFeatureList feature_list;
+  feature_list.InitAndEnableFeature(brave_shields::features::kAdblockOnlyMode);
+
+  BravePrefProvider provider(
+      testing_profile()->GetPrefs(), false /* incognito */,
+      true /* store_last_modified */, false /* restore_session */);
+  GURL url("https://brave.com/");
+
+  ShieldsCookieSetting shields_cookie_settings(&provider,
+                                               testing_profile()->GetPrefs());
+  CookieSettings cookie_settings(&provider);
+  ShieldsFingerprintingSetting fp_settings(&provider);
+  ShieldsScriptSetting script_settings(&provider);
+  ShieldsHttpsUpgradeSetting https_upgrade_settings(&provider);
+
+  shields_cookie_settings.SetPreMigrationSettings(
+      ContentSettingsPattern::FromURL(url), CONTENT_SETTING_BLOCK);
+  cookie_settings.SetPreMigrationSettings(ContentSettingsPattern::FromURL(url),
+                                          CONTENT_SETTING_BLOCK);
+  fp_settings.SetPreMigrationSettings(ContentSettingsPattern::FromURL(url),
+                                      CONTENT_SETTING_BLOCK);
+  script_settings.SetPreMigrationSettings(ContentSettingsPattern::FromURL(url),
+                                          CONTENT_SETTING_BLOCK);
+  https_upgrade_settings.SetPreMigrationSettings(
+      ContentSettingsPattern::FromURL(url), CONTENT_SETTING_BLOCK);
+
+  testing_profile()->GetPrefs()->SetBoolean(
+      brave_shields::prefs::kAdblockAdBlockOnlyModeEnabled, true);
+
+  // Verify that adblock only mode overrides settings.
+  shields_cookie_settings.CheckSettingsWouldAllow(url);
+  cookie_settings.CheckSettingsWouldAllow(url);
+  fp_settings.CheckSettingsWouldAllow(url);
+  script_settings.CheckSettingsWouldAllow(url);
+  https_upgrade_settings.CheckSettingsWouldAsk(url);
+
+  provider.ShutdownOnUIThread();
+}
+
+TEST_F(BravePrefProviderTest,
+       AdblockOnlyModeContentSettings_DefaultValues_OffTheRecord) {
+  base::test::ScopedFeatureList feature_list;
+  feature_list.InitAndEnableFeature(brave_shields::features::kAdblockOnlyMode);
+
+  BravePrefProvider provider(
+      testing_profile()->GetPrefs(), true /* incognito */,
+      true /* store_last_modified */, false /* restore_session */);
+  GURL url("https://brave.com/");
+
+  ShieldsCookieSetting shields_cookie_settings(&provider,
+                                               testing_profile()->GetPrefs());
+  CookieSettings cookie_settings(&provider);
+  ShieldsFingerprintingSetting fp_settings(&provider);
```

Comment:
Lets not use abbreviations for readability.

Reply:
This is consistent with naming in other tests in the file.

Reply:
In my opinion we should move to the correct way, and raise a PR to cleanup.

Reply:
I think consistency is better and safer now. It will be simpler for followup cleanup PR to fix the same wrong pattern through the file. 
If we fix it now, than we will have two patterns and there is a chance that followup PR will change an old part inconsistently.

Opened issue follow-up: https://github.com/brave/brave-browser/issues/48635